src/Workspaces/Core/Portable/Workspace/Host/SyntaxTreeFactory/AbstractSyntaxTreeFactoryService.cs

```diff
@@ -16,22 +16,11 @@ namespace Microsoft.CodeAnalysis.Host
 {
     internal abstract partial class AbstractSyntaxTreeFactoryService : ISyntaxTreeFactoryService
     {
-        private readonly Lazy<int> _minimumLengthForRecoverableTree;
-
         internal SolutionServices SolutionServices { get; }
 
         public AbstractSyntaxTreeFactoryService(SolutionServices services)
         {
             SolutionServices = services;
-
-            // Create this lazily; this ultimately needs to read options under the covers, which isn't necessary until
-            // we are actually parsing stuff. This moves some extra loading of MEF parts and services out of the solution load
-            // path.
-            _minimumLengthForRecoverableTree = new Lazy<int>(() =>
-            {
-                var cacheService = SolutionServices.GetService<IProjectCacheHostService>();
-                return (cacheService != null) ? cacheService.MinimumLengthForRecoverableTree : int.MaxValue;
```

Comment:
this would now always return int.MaxValue in either path, so it isn't needed to compute.  _minimumLengthForRecoverableTree is then always int.MaxValue, so it was removed and inlined.